apps/trading/specs/index.spec.tsx

```diff
@@ -1,26 +1,18 @@
 import React from 'react';
 import { render } from '@testing-library/react';
-
 import Index from '../pages/index.page';
-import { VegaWalletContext } from '@vegaprotocol/wallet';
+
+jest.mock('@vegaprotocol/ui-toolkit', () => {
+  const original = jest.requireActual('@vegaprotocol/ui-toolkit');
+  return {
+    ...original,
+    AgGridDynamic: () => <div>AgGrid</div>,
+  };
+});
 
 describe('Index', () => {
   it('should render successfully', () => {
-    const { baseElement } = render(
-      <VegaWalletContext.Provider
-        value={{
-          keypair: null,
-          keypairs: null,
-          connect: jest.fn(),
-          disconnect: jest.fn(),
-          selectPublicKey: jest.fn(),
-          connector: null,
-          sendTx: jest.fn(),
-        }}
-      >
-        <Index />
-      </VegaWalletContext.Provider>
-    );
+    const { baseElement } = render(<Index />);
     expect(baseElement).toBeTruthy();
```

Comment:
As per https://vegaprotocol.slack.com/archives/C8WS7M50E/p1647868686724769 we should move to use `screen` where possible. As long as `<Index />` can accept props then I think this should work nicely:

```suggestion
    render(<Index data-testid='render-test' />);
    expect(screen.getByTestId('render-test')).toBeInTheDocument();
```